regression/esbmc-solidity/return_4/contract.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.5.0;
+
+// Ensure functions can return different type of value.
+contract A {
```

Comment:
What's the purpose of this test case?

Reply:
My mistake, this should test for member calls of other integer return types. I will add the member calls assertion below.